Unity/Assets/Scripts/json/JsonManager.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+// using System.Collections;
+// using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+
+
+//class名要検討 どんなデータなのか分かる名前だとよい
+[Serializable]
+public class JsonData
+{
+    public Status[] statuses;
+    public Tag[] tags;
+}
+
+[Serializable]
+public class Status
+{
+    public int id;
+    public string name;
+    public Goal[] goals;
+    public string updatedAt;
+}
+
+[Serializable]
+public class Goal
+{
+    public int id;
+    public string name;
+    public string description;
+    public string createdAt;
```

Comment:
```suggestion
    public string createdAt;　// YYYY-MM-DD/HH:MM:SS
```
といった具合でデータの不定な日時表現を表した方が保守性は上がりそうです。